### ISSUE TITLE
No IP, TXT records or resolved events

### ISSUE THREAD

**Issue by TomMettam** [becvert/cordova-plugin-zeroconf#77]
On Android, for most devices, we get "added" event for most devices, however this event is missing the IP address and TXT record data.

avahi-browse shows all the devices correctly with full info.

**Comment by emcniece** [comment]
The `resolved` event is the one for detecting addresses - do your expected details show up in there?

**Comment by TomMettam** [comment]
No, as mentioned above we don't receive the resolved events at all unless we call reInit first.

**Comment by emcniece** [comment]
Wondering if this is a timing issue. Do you kick off your `watch()` from inside (or after) a `platform.ready()` or similar call?

**Comment by TomMettam** [comment]
Yes,  inside the platform.ready callback. I thought timing issue too, so i tried a ten second timer instead.  No dice

**Comment by emcniece** [comment]
Intriguing, thanks for confirming. Some more thoughts...

- Mobile data enabled on the Android devices? Does disabling mobile data change anything?
- Anything interesting in `adb` logs (either in terminal or Android Studio) pertaining to network status or this plugin?
- You say "for most devices" - are there specific devices that are reliable, or does it vary on each app boot?

**Comment by TomMettam** [comment]
* No mobile data,  wifi only. 

* Nothing interesting that I could see.  Network connectivity was solid and I could see the query and responses with wireshark.

* Only one device came through as "resolved" (and actually this event fired repeatedly). It was a synology NAS device which was itself performing mdns queries. 

After issuing reInit all the other devices on the network came through as resolved.

**Comment by tonetechnician** [comment]
Picking up on a relatively inactive thread here, but I'm also running into this issue with iOS.

Particularly, I notice the issue doesn't happen when creating the bonjour service on my Mac - if so then the iOS device picks up the resolved action just fine. But when the service is created on on Windows, I'm only left with the "added" action, and it never gets resolved. 

Did you guys figure it out?
